02_virtualization_and_containers/virtualmachines_containers_exercise.md

```diff
@@ -162,43 +205,43 @@ We want to provision (`config.vm.provision`) the box in several steps. You can r
 
 #### 5. Creating a Merge Request (Vagrant)
 
-- After checking your Vagrant box carefully, please open a merge request in the GitHub Repository ["Exercise Virtual Machines"](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering/exercise-virtual-machines):
+- After checking your Vagrant box carefully, please open a merge request in the GitHub Repository ["Exercise Virtual Machines"](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2223/exercise-virtual-machines):
     - As title, choose "[`USERNAME`] Vagrant Box Provisioning".
     - Make sure all files are up to date (`testfile`, `bootstrap.sh`, `Vagrantfile`). Do not add the hidden folder `.vagrant` to Git.
     - Attach the screenshot ("Attach a file") that you made in the previous step.
-    - Add the label `Vagrant` label to the issue and assign the issue to `jaustar`.
+    - Add the label `Vagrant` label to the issue and assign the issue to `desaiin`.
     - Double-check that all files are in the repository and up to date.
     - Double-check that the source (the new branch on your fork) and the target branch (`main` of original repo) of the merge request is correct.
     - If everything looks good, create the merge request.
 
 ### Further Information (Vagrant)
 
 - You might want to add the option `--provision` to the `vagrant up` or use `vagrant provision` command if you want to reprovision a running box (without destroying it first). In case the box is not being build/rebuild, please read the [documentation about provisioning](https://www.vagrantup.com/docs/provisioning) carefully.
-- By default, Vagrant will store some larger files like the base images/boxes etc.\ in `${HOME}/vagrant.d`. If Vagrant should use a different directory, you can set the environment variable `VAGRANT_HOME` to point the alternative directory. This could look like this:
+- By default, Vagrant will store some larger files like the base images/boxes etc.\ in `${HOME}/vagrant.d`. If Vagrant should use a different directory, you can set the environment variable `VAGRANT_HOME` to point to the alternative directory. This could look like
```

Comment:
```suggestion
- By default, Vagrant will store some larger files like the base images/boxes etc.\ in `${HOME}/.vagrant.d`. If Vagrant should use a different directory, you can set the environment variable `VAGRANT_HOME` to point to the alternative directory. This could look like
```